lib/Doctrine/DBAL/Driver/OCI8/OCI8Statement.php

```diff
@@ -134,7 +139,15 @@ static public function convertPositionalToNamedPlaceholders($statement)
      */
     public function bindValue($param, $value, $type = null)
     {
-        return $this->bindParam($param, $value, $type, null);
+        if ($type === \PDO::PARAM_LOB || -99 === $type) {
+            $descriptor = oci_new_descriptor($this->_dbh, OCI_D_LOB);
+            $descriptor->writeTemporary($value, $type === -99 ? OCI_TEMP_CLOB : OCI_TEMP_BLOB);
```

Comment:
The [PHP docs](http://php.net/manual/en/oci-lob.writetemporary.php) are encouraging us to close the descriptor when finished. Maybe we should do that?